features/pageobjects/basic_auth.page.js

```diff
@@ -2,14 +2,14 @@
 // import Page from "./page.js";
 
 class BasicAuthPage {
-  get message() {
-    return $(".example > p");
+  get body() {
+    return $("body");
```

Comment:
I understand this is the [complete opposite of best practices per wdio](https://webdriver.io/docs/selectors), but I'm working with what we are given here on "The Internet". The basic auth page only returns `<html><head></head><body></body></html>` when using a headless browser and submitting incorrect credentials, and only does so if you run headlessly.

If you run with a headed browser, it doesn't even return the "Not authorized" HTML body--just an empty string HTTP response and the browser will re-prompt you to submit credentials until you click cancel, and I [couldn't find any way](https://webdriver.io/search?q=basic%20auth) for wdio to interact with the basic auth dialog to cancel/dismiss it.